aws-alb-ingress-controller/templates/pdb.yaml

```diff
@@ -1,13 +1,12 @@
-{{ $root := . }}
-{{- with .Values.podDisruptionBudget }}
+{{- if .Values.podDisruptionBudget }}
 apiVersion: policy/v1beta1
 kind: PodDisruptionBudget
 metadata:
-  name: {{ template "aws-alb-ingress-controller.fullname" $root }}
+  name: {{ template "aws-alb-ingress-controller.fullname" . }}
   namespace: {{ .Release.Namespace }}
-  labels: {{ include "aws-alb-ingress-controller.labels" $root | nindent 4 }}
+  labels: {{ include "aws-alb-ingress-controller.labels" . | nindent 4 }}
 spec:
+  {{- toYaml .Values.podDisruptionBudget | nindent 2 }}
   selector:
-    matchLabels: {{ include "aws-alb-ingress-controller.instance-labels" $root | nindent 6 }}
-{{ toYaml . | indent 2 }}
+    matchLabels: {{ include "aws-alb-ingress-controller.instance-labels" . | nindent 6 }}
```

Comment:
@cw-sakamoto 
new line?

Reply:
oh...

Reply:
There are similar parts in other templates, so I'll fix them together in the next PR...